certified-connectors/Tulip/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@eddya101 Settings.json is not required in open source repository. Please remove this file from the pull request. You can submit this file later in ISV Studio.